crates/miden-lib/src/transaction/memory.rs

```diff
@@ -41,10 +41,11 @@ pub type StorageSlot = u8;
 // | Num procedures    | 28 (7)                                | 31 (7)                              |                                     |
 // | Procedures info   | 32 (8)                                | 2_079 (519)                         | 255 procedures max, 8 elements each |
 // | Padding           | 2_080 (520)                           | 2_083 (520)                         |                                     |
-// | Num storage slots | 2_084 (521)                           | 2_087 (521)                         |                                     |
-// | Storage slot info | 2_088 (522)                           | 4_127 (1031)                        | 255 slots max, 8 elements each      |
-// | Initial slot info | 4_128 (1032)                          | 6_167 (1541)                        | Only present on the native account  |
-// | Padding           | 6_168 (1541)                          | 8_191 (2047)                        |                                     |
+// | Proc tracking     | 2_084 (521)                           | 2_339 (584)                         |                                     |
```

Comment:
nit: I would maybe add a comment that this is for 255 procedures, 1 element each.